src/mono/sample/wasm/browser-profile/README.md

```diff
@@ -13,16 +13,18 @@ public static void StopProfile(){}
 2. Initialize the profiler in the main javascript (e.g. main.js)
 
 ```
-await createDotnetRuntime({
-    onConfigLoaded: () => {
-        if (config.enableProfiler) {
-            config.aotProfilerOptions = {
-                write_at: "<Namespace.Class::StopProfile>",
-                send_to: "System.Runtime.InteropServices.JavaScript.JavaScriptExports::DumpAotProfileData"
+await dotnet
+    .withModuleConfig({
+        onConfigLoaded: () => {
```

Comment:
could be replaced by `withConfig({"enableProfiler":{"write_at":"etc"}})`

I wonder if I renamed `writeAt`

Reply:
Yea, you did.